src/outputs/parthenon_hdf5.cpp

```diff
@@ -276,40 +273,40 @@ void PHDF5Output::WriteOutputFile(Mesh *pm, ParameterInput *pin, SimTime *tm) {
 
   int max_level = pm->GetCurrentLevel() - pm->GetRootLevel();
   if (tm != nullptr) {
-    status = writeH5AI32("NCycle", &(tm->ncycle), file, localDSpace, myDSet);
-    status = writeH5AF64("Time", &(tm->time), file, localDSpace, myDSet);
+    writeH5AI32("NCycle", &(tm->ncycle), file, localDSpace, myDSet);
+    writeH5AF64("Time", &(tm->time), file, localDSpace, myDSet);
   }
```

Comment:
Same with `status` as `ierr`. We should check that it's a success and die if it's not, rather than just not looking.